internal/chmod_sock.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import (
+	"log"
+	"os"
+)
+
+// UpdateSocketFilePermissions set socket file permissions
+func UpdateSocketFilePermissions(sockFileName string) {
```

Comment:
Already done in `ManualListener`, we only need differentvalue for permissions

Reply:
What is the use case (purpose) of `ManualListener`?

Reply:
`ManualListener` is for running non-systemd env e.g. Docker. But in this case it is not relevant except that it also does `Chmod`